src/main/java/kr/codesqaud/cafe/domain/Article.java

```diff
@@ -0,0 +1,34 @@
+package kr.codesqaud.cafe.domain;
+
+public class Article {
+    private final String writer;
+    private final String title;
+    private final String content;
+    private int index;
```

Comment:
Lon
```suggestion
    private Long Id;
```
관행적으로 Long 타입 Id로 순번을 저장하는 경우가 많습니다.